paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/procedure/ExpirePartitionsProcedure.java

```diff
@@ -75,21 +82,45 @@ public String identifier() {
             String timestampFormatter,
             String timestampPattern,
             String expireStrategy,
-            Integer maxExpires)
+            Integer maxExpires,
+            String options)
             throws Catalog.TableNotExistException {
-        FileStoreTable fileStoreTable = (FileStoreTable) table(tableId);
+        Map<String, String> dynamicOptions = new HashMap<>();
```

Comment:
These code is same as three ExpirePartitionsProcedure

Reply:
Thks, added a unified util method.